### PR TITLE
Add update.json for jquery.mixitup

### PR DESCRIPTION
**Pull request by simon-thorpe**

### CODE REVIEW

File: files/jquery.mixitup/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "mixitup",
+  "repo": "patrickkunka/mixitup",
+  "files": {
+    "include": ["jquery.mixitup.js", "jquery.mixitup.min.js"]
```
Comment (jimaek) [inline files/jquery.mixitup/update.json]: There are no such files here https://github.com/patrickkunka/mixitup/tree/v2.1.11
The bot will fail

**Comment by simon-thorpe** [comment]
Sorry about that. I've added the correct basePath. While I was there I removed the non-minified version as this seems preferred.

**Comment by jimaek** [comment]
Do the following:
```
     "include": ["build/jquery.mixitup.min.js", "src/jquery.mixitup.js"]
```

This way we wont break the file structure between minor versions.

**Comment by simon-thorpe** [comment]
That should work now.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**